src/Core/Prefab/GlobeView.js

```diff
@@ -281,6 +281,9 @@ GlobeView.prototype = Object.create(View.prototype);
 GlobeView.prototype.constructor = GlobeView;
 
 GlobeView.prototype.addLayer = function addLayer(layer) {
+    if (!layer) {
```

Comment:
Are these check really necessary ?
(if they are, you should add them to View.addLayer and PanoramaView.addLayer as well)

Reply:
Also, this breaks current code: now this function can return undefined (it couldn't before). Please consider returning a rejected promise with a suitable `Error` object.

Reply:
yes it is necessary because this error is not handle
I added error handling in all `addLayer`

Reply:
At the same time, is it really necessary to do so? If we let the code like it is before, users would have a rejected promise with "Error cannot read "type" of undefined" or something like that, which is imho clear enough. Anyway, why not having a better error message.